src/realm/object-store/c_api/error.cpp

```diff
@@ -78,7 +79,7 @@ static realm_error_t* create_error(std::exception_ptr* ptr)
                 // copy the message. rethrow_exception copies the exception object on some platforms (Windows) hence
                 // the ex.what() ptr will get invalidated
                 std::string message(ex.what());
-                char* messagePtr = static_cast<char*>(std::calloc(message.size() + 1u, sizeof(char*)));
+                char* messagePtr = static_cast<char*>(std::malloc(message.size() + 1u));
```

Comment:
thanks, will change it

Reply:
is there a way not to zero initialize this array like what `std::make_unique<char[]>` will do

Reply:
Why would you not want to zero initialize the array?

Reply:
we are zeroing it and then immediately coping data, fully overwriting the whole array.

Reply:
imo a perfect case for using malloc

Reply:
Since we are not on a hot path here, I left the smart pointers version which zeros the array which could be skipped but it's cleaner than the malloc I used.